### ISSUE TITLE
Make resizable mode of 2005 Gameframe (buttons) look like the fixed mode.

### ISSUE THREAD

**Issue by AmazingJesus** [runelite/runelite#7365]
The resizable mode uses 2005 icons but 2007 background/border.

Current: 
https://i.gyazo.com/6a53bf1e746e4bcec13499b8e04f5973.png

How it should be:
https://i.gyazo.com/eff17de994d0ca07b2716b315f4ccee6.png

Blank version:
https://i.gyazo.com/529cdf3764daf49ffd6a4c4beceb7fc4.png

**Comment by deathbeam** [comment]
That looks horrible tbh

**Comment by troypressley1** [comment]
Makes sense but I'm not sure it can be done really since all blocks are different sizes. I'd agree with deathbeam in general, I love the 2005 interface, but the resizable one looks a bit strange

**Comment by AmazingJesus** [comment]
@deathbeam It looks worse to have 2 different gameframes mixed up tho... Can't have 2005 icons with 2007 background
